comment.go

```diff
@@ -0,0 +1,31 @@
+// Licensed under the MIT license, see LICENCE file for details.
+
+package quicktest
+
+import "fmt"
+
+// Commentf returns a test comment whose output is formatted according to
+// the given format specifier and args. For instance:
+//
+//     c.Assert(a, qt.Equals, 42, qt.Commentf("answer is not %d", 42))
+//
+// The provided extra information if printed when the test fails.
+func Commentf(format string, args ...interface{}) Comment {
+	return Comment{
+		format: format,
+		args:   args,
+	}
+}
+
+// Comment represents a comment on a test failure, and is used to provide extra
```

Comment:
This doc comment is arguably misleading, because it sounds like the extra information contributes to the checks or assertions.

    // Comment represents additional information on a check or an assertion which is
    // displayed when the check or assertion fails.

Reply:
Done.